systemd/systemd.go

```diff
@@ -204,6 +204,14 @@ func MockJournalctl(f func(svcs []string, n int, follow bool) (io.ReadCloser, er
 	}
 }
 
+type MountUnitOptions struct {
+	// Whether the unit is transient or persistent across reboots
+	Lifetime                                UnitLifetime
+	SnapName, Revision, What, Where, Fstype string
```

Comment:
nitpick, I think it's easier to read if these were all on their own line

```suggestion
	SnapName string
	Revision string
	What string
	Where string
	Fstype string
```

Reply:
Done!